uiowa/uiowa
# Remove duplicate page bundle class

Resolves #7495

# To Test

```
ddev blt ds --site=brand.uiowa.edu && ddev drush @brand.local uli admin/content/sitenow-converter
```
No import or sync errors.
You should be able to get to this page without error and be able to convert a page to v3.

```
ddev blt ds --site=sandbox.uiowa.edu && ddev drush @sandbox.local uli node/2211
```

No import or sync errors.
You should see rendered page list blocks and you should be able to add another one.
